WebDriverAgentRunner/Info.plist

```diff
@@ -20,5 +20,11 @@
 	<string>????</string>
 	<key>CFBundleVersion</key>
 	<string>1</string>
+	<key>NSLocationAlwaysAndWhenInUseUsageDescription</key>
+	<string></string>
```

Comment:
is it allowed for these messages to be empty?

Reply:
Yes. The keys are needed.
I tested without these keys, but my env worked. But official documentation and some articles addressed these keys, so I've added them for safe.